README.md

```diff
@@ -63,6 +63,10 @@ To launch the game with the patched EBOOT, open up RPCS3, go to File, Boot SELF/
 Assuming you are running the patched version of RPCS3, you patched the file correctly, the database is migrated, and
 Lighthouse is running, the game should now connect.
 
+### LittleBigPlanet 1
+
+For LittleBigPlanet 1 to work with RPCS3, follow the steps for LittleBigPlanet 2. First, download [HXD](https://mh-nexus.de/downloads/HxDSetup.zip). Once HXD is set up, open your `EBOOTlocalhost.elf` file with HXD and click on Search -> Replace and click on the `Hex-values` tab. In the `Search for` section put `73 63 65 4E 70 43 6F 6D 6D 65 72 63 65 32`, then in the `Replace with` section put `00 00 00 00 00 00 00 00 00 00 00 00 00 00`, and then click `Replace all`. After that go to File -> Save.
```

Comment:
Can you explain what the patch does in a little bit more detail?

Reply:
Basically it patches out the reference to sceNpCommerce2 in hexadecimal, since when patching it out in plain text, it doesn't remove the reference but instead replace is it with periods.

Reply:
Oh, I meant in the readme.md.

Reply:
I've fixed the README.md